### ISSUE TITLE
leveraging the nebual-java session pool

### ISSUE THREAD

**Issue by wey-gu** [nebula-contrib/ngbatis#72]
are we going to use nebula-java session pool in ngbatis?

**Comment by CorvusYe** [comment]
If the versions are functionally compatible, it is a better choice to directly use the nebula-java's session pool.

**Comment by wey-gu** [comment]
> If the versions are functionally compatible, it is a better choice to directly use the nebula-java's session pool.

Yes, the 3.3.0 nebula-java introduced this by @Nicole00, which is(for thrift level perspective) compatible with all v3.x.y nebulagraph :)
